README.md

```diff
@@ -242,16 +208,38 @@ https://wapo-testnet.phala.network/ipfs/<cid>?key=<key>
 
 </details>
 
+#### Accessing Secrets in Your Code
+To access your secrets you will have to get them from secret `vault` via `process.env.secret`. Here is an example of accessing the secret key value of `secretSalt`.
+```typescript
+let vault: Record<string, string> = {}
+try {
+  vault = JSON.parse(process.env.secret || '')
+} catch (e) {
+  console.error(e)
+  return c.json({ error: "Failed to parse secrets" })
+}
+const secretSalt = (vault.secretSalt) ? vault.secretSalt as string : 'SALTY_BAE'
+```
+
 #### Example:
 - [Derived ECDSA Key](https://wapo-testnet.phala.network/ipfs/QmVJ3xknfRevUkc68iZc4RdPSLL2gLD8WagwMQCdGMyC4g?key=6978ea391960e272)
 - [Sign 'signedByTEE'](https://wapo-testnet.phala.network/ipfs/QmVJ3xknfRevUkc68iZc4RdPSLL2gLD8WagwMQCdGMyC4g?key=6978ea391960e272&type=sign&data=signedByTEE)
 - [Verify Signature](https://wapo-testnet.phala.network/ipfs/QmVJ3xknfRevUkc68iZc4RdPSLL2gLD8WagwMQCdGMyC4g?key=6978ea391960e272&type=verify&data=signedByTEE&signature=0x9b1b413f1573da2ada426f4da99416b6081ef7246a23990c0c13d764f566083920a4b07636b7d7a582fbc3d98ad42bedc26410764a4cd2963058792121a5d63d1b)
 - [Send TX on Base Sepolia](https://wapo-testnet.phala.network/ipfs/QmVJ3xknfRevUkc68iZc4RdPSLL2gLD8WagwMQCdGMyC4g?key=6978ea391960e272&type=sendTx&to=0xC5227Cb20493b97bb02fADb20360fe28F52E2eff&gweiAmount=420)
 
-### Access Queries
-To help create custom logic, we have an array variable named `queries` that can be accessed in the `Request` class. To access the `queries` array variable `type` value at index `0`, the syntax will look as follows:
+### Handling Requests
+Check the [`Hono` docs](https://hono.dev/docs) for information on accessing URL `queries` or `body` from a `post` request.
+
+> **NOTICE**
+>
+> The current implementation of `hono` is limited. For example, routing paths are not supported, yet. Try to only use the `/` path & utilize [URL queries](https://hono.dev/docs/api/request#query) or pass data via the [body](https://hono.dev/docs/api/request#json) in a post request
+
 ```typescript
-const type = req.queries.type[0] as string;
+// Access query value for a URL query named `type`
+let queries = c.req.queries() || {}
+const getType = (queries.type) ? queries.type[0] as string : ''
+// Access body from post request
+const data = await c.req.json()
 ```
```

Comment:
I'm considering using zod for typed input validation, which is already supported but I didn't mentioned that before. Any ideas?
https://hono.dev/examples/zod-openapi